packages/semcom-core/docs/modules/specifications/pages/637440573-support-registering-components.adoc

```diff
@@ -0,0 +1,93 @@
+== Technical Specification: Support registering of components
+
+=== Author(s)
+
+* Stijn Taelemans
+
+=== References
+
+* Functional analysis: see semcom-core docs
+
+* Branch: `637440573-support-registering-components`
+* Projects:
+** https://github.com/digita-ai/semcom[semcom]
+
+=== Introduction
+
+==== Overview
+
+This specification is about part of the SDK that registers the Components on the web page using the `CustomElementRegistry`.
+
+=== Solution
+
+_This is a proposed solution and is open to change. If you feel some things can be done better, please discuss with this document's link:###authors[author(s)]._
+
+==== semcom-core
+
+===== Models
+
+====== ComponentMetadata
+
+Add the `tag: string` attribute to the existing `ComponentMetadata` model. We'll need it later for registering the component.
+
+Example: 
+
+[source, json]
+----
+{
+    "uri": "https://components.digita.ai/album",
+    "name": "Image Album Component",
+    "label":  "Component that renders images",
```

Comment:
What is the difference between a component's name and label?

Reply:
changed to label and description